### DISCUSSION TITLE
Rosetta protein-protein dock local

### DISCUSSION THREAD

**Discussion by Vini161** [RosettaCommons/rosetta#132] [Rosetta]
I want to dock two proteins and I am aware of their relative positions through literature. However there is no native structure. Is this fine? I can use further advice.
I have AB.pdb, CD.pdb and AB_CD_complex.pdb

**Comment by roccomoretti** [comment]
If you're lacking crystal structures for the inputs, Rosetta protein-protein docking doesn't really care where the input monomers are from. You can use crystal structures of the monomers, or you can use structures from a different computational modeling pipeline (e.g. Rosetta homology modeling or a machine learning structure prediction)

If you're talking about lacking a "native structure" of the complex, that's actually the normal state of things -- the reason to do docking is because you're trying to figure out the structure of the complex. Most Rosetta protocols take the reference "native" structure because they're set up to make benchmarking easier -- if you _do_ know the structure of the native complex, you can gauge how well Rosetta is working by comparing the output to the known "true" structure. But that's only a convenience function. Pretty much any Rosetta protocol which can take a native reference protein should be robust to simply omitting the `-native` option if you don't have one. That will simply bypass the compute-comparison-to-native steps in the protocol, but should work identically otherwise.

**Comment by Vini161** [comment]
Hi, I had another issue with local docking. I have aligned my strcutures to pre-existing incomplete crystal bound structure and I want very less perturbation,  using 1 A 1 degree dock_pert still gives large movement when compared to original. Does just using local refine, ie. high res docking_local_refine in any way affect the  assumptions of docking protocol, so that I don't move the centrod. I  am basically making structres by introducing very less rotation-2-6 degrees and comparing each, I have the one with 0 degree rotation wrt to native now?

**Comment by roccomoretti** [comment]
One thing to consider is "movement with respect to where?" -- Depending on the precise setup, docking may adjust things such that rotations are centered around the protein-protein interface. So small (e.g ~1 degree) changes in the orientation may be compounded when they reach the other side of the protein. When examining the amount of perturbation in docking, it's often best to pay more attention to the area around the interface, rather than how much movement you have on the far side of the protein.

The other thing to keep in mind is that the amounts of perturbation may be setting the step size in the Monte Carlo process. So instead of "1 Ang" being the maximum (or average) amount of movement, Rosetta jitters the structure randomly in the 0-1 Ang range (or often with a normally distributed perturbation with a standard deviation of 1 Ang), checks the energy and then repeats the process for hundred of times. As such, you may get more movement than you'd naively anticipate, particularly if it's moving to an energy well.
